media/css/mozorg/home/includes/pocket-promo.scss

```diff
@@ -0,0 +1,42 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+.t-pocket-promo {
+    background-color: #FFF0E3db;
```

Comment:
Edge support for the 8 digit hex is relatively recent (2020). 

If the background colour doesn't display it's still readable though so I guess I'm writing all this out to not actually ask for any changes.

Reply:
I copied it from the Figma file, but updated to be a 6-digit hex to make sure it displays on older browsers 👍🏼